packages/dds/shared-object-base/src/sharedObject.ts

```diff
@@ -762,15 +766,13 @@ export abstract class SharedObject<
 		return this._serializer;
 	}
 
-	/**
-	 * @param id - The id of the shared object
```

Comment:
The docs for these are inherited automatically from `SharedObjectCore`. We don't need to repeat them here.